src/views/RichWorkspace.vue

```diff
@@ -134,9 +135,12 @@ export default {
 			})
 		},
 		getFileInfo(autofocus) {
-			this.loaded = false
-			this.autofocus = false
+			if (!this.hasRichWorkspace) {
+				return
+			}
 			this.ready = false
+			this.loaded = true
+			this.autofocus = false
 			const params = { path: this.path }
```

Comment:
FIXME: We still need to drop the following api request if we can get all info from the webdav response of the file list

Reply:
I kept this for now but instead made the loading more smooth as we have the markdown content already we can just preview that